src/test/java/com/palehorsestudios/alone/player/PlayerTest.java

```diff
@@ -364,18 +364,75 @@ public void improveShelter() {}
 
   @Test
   public void gatherFirewood() {
+    Result gatherFirewoodResult = player.gatherFirewood();
+    int firewoodAmount = gatherFirewoodResult.getFirewood();
+    double caloriesBurned = gatherFirewoodResult.getCalories();
+    int addedMorale = gatherFirewoodResult.getMorale();
+    if (firewoodAmount == 1 ) {
+      assertEquals(-75.0, caloriesBurned, 0.01);
+      assertEquals(Math.ceil(firewoodAmount / 2.0), addedMorale, 0.01);
+    }
+    else if (firewoodAmount == 3) {
+      assertEquals(-150.0, caloriesBurned, 0.01);
+      assertEquals(Math.ceil(firewoodAmount / 2.0), addedMorale, 0.01);
+    }
+    else if (firewoodAmount == 5) {
+      assertEquals(-300.0, caloriesBurned, 0.01);
+      assertEquals(Math.ceil(firewoodAmount / 2.0), addedMorale, 0.01);
+    }
+    assertEquals("Good Job! You just gathered " + firewoodAmount + " bundles of firewood.", gatherFirewoodResult.getMessage());
+
     // TODO: Need to make this test more thorough
-    System.out.println(player.gatherFirewood().getMessage());
   }
 
   @Test
-  public void getWater() {}
+  public void getWater() {
+    Result getWaterResult = player.getWater();
```

Comment:
We should test the updated Player weight. This can wait until after this PR is merged. I added constants for player weight for each activity level and success level in the code I will be pushing up.